frontend/pokino-app/src/app/view/main-menu.component.html

```diff
@@ -3,6 +3,19 @@
 		<img class="pokemon-background" src="../../assets/images/pokemons/squirtle.png"/>
 	</div>
 	<div class="container">
+		<span class="userNameText">Your Name: </span>
```

Comment:
nur kurz für mein verständniss - wenn der player den name eingibt, machst du auch einen call ans backend? unter dem game/login endpoint? weil der gibt dir zurück, ob der name verfügbar ist oder nicht.

Reply:
jop, in der main-menu.component.ts ab line 108